Challenge/src/components/ListOfCrypto.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from "react";
+import { FlatList, Text, View, Image, StyleSheet } from "react-native";
+import list from "./list";
+
+/* export default function List(){
+
+    const [list, setList] = useState()
+
+  useEffect (()=> {
+   async function Data() {
+    const response = await fetch('https://data.messari.io/api/v1/assets');
+    const json = await response.json();
+     setList(json.data)
+   } 
+   Data();
+}, [])  
+} */
+
+const ListOfCurrencies = () => {
+    return (
+        <View style={styles.container}>
+            {list.map(c => (
+                <>
+                    <View style={styles.crypto} key={c.id}>
+                        <View style={styles.crypoProfile}>
+                            <Image
+                                style={styles.image}
+                                source={c.img}
+                            />
+                            <View style={styles.cryptoInfo}>
+                                <Text style={styles.name}>{c.name}</Text>
+                                <Text style={styles.symbol}>{c.symbol}</Text>
+                            </View>
+                        </View>
+                        <View style={styles.cryptoPrice}>
+                            <Text style={styles.name}>${c.price}</Text>
+                            <View style={styles.cryptoPercent}>
+                                <Image source={c.percent > 0 ? c.arrowGreen : c.arrowRed} />
+                                <Text style={[styles.percentContainer, c.percent > 0 ? styles.colorGreen : styles.colorRed]}>{c.percent}%</Text>
```

Comment:
you can use styled-components to declare a custom component and send it the validation value, then change the color inside the style